flixel/addons/editors/tiled/TiledImageSet.hx

```diff
@@ -0,0 +1,51 @@
+package flixel.addons.editors.tiled;
+
+import haxe.xml.Fast;
+
+/**
+ * Copyright (c) 2013 by Samuel Batista
+ * (original by Matt Tuttle based on Thomas Jahn's. Haxe port by Adrien Fischer)
+ * This content is released under the MIT License.
+ */
+class TiledImageSet implements Dynamic<String>
```

Comment:
I was not really sure what "Dynamic<String>" is so I made another class just in case. It is better if I use the same class and add another function to add the image properties?